find/client/option.go

```diff
@@ -7,6 +7,7 @@ import (
 
 type config struct {
 	httpClient *http.Client
+	dhstoreUrl string
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
	dhstoreURL string
```